src/Generator.Rewrite/Options.cs

```diff
@@ -0,0 +1,24 @@
+﻿using CommandLine;
+using CommandLine.Text;
+
+namespace OpenTK.Rewrite
+{
+    public class Options
+    {
+        [Option('i', "assembly", Required = true,
```

Comment:
I assume "i" for "input". Can it be consistent and be "a" though? I'd much sooner guess the first letter of "assembly" than I would "i" for input.

Reply:
Do these have to be single letter identifiers?

Reply:
@Frassle Yeah, that's reasonable. I'll change it.
@daerogami Yes, in line with the POSIX standard. You have a short identifier that is one character, and an optional longer identifier with an unlimited number of characters.